packages/shared/prisma/schema.prisma

```diff
@@ -422,6 +426,36 @@ enum ScoreSource {
     EVAL
 }
 
+model AnnotationConfig {
+    id           String        @id @default(cuid())
+    createdAt    DateTime      @default(now()) @map("created_at")
+    updatedAt    DateTime      @default(now()) @updatedAt @map("updated_at")
+    projectId    String        @map("project_id")
+    project      Project       @relation(fields: [projectId], references: [id], onDelete: Cascade)
+    name         String
+    dataType     ScoreDataType
+    isArchived   Boolean       @default(false) @map("is_archived")
+    minValue     Float?        @map("min_value")
+    maxValue     Float?        @map("max_value")
+    categories   Json?         @map("categories")
+    authorUserId String?       @map("author_user_id")
```

Comment:
do we need author of an annotation config? we can cover edit history via the auditlog table

Reply:
I've removed it 👍